auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/AuthenticationExceptionExtensions.kt

```diff
@@ -3,32 +3,33 @@ package com.auth0.auth0_flutter
-    return mapOf(
-        "_statusCode" to this.statusCode,
-        "_errorFlags" to mapOf(
-            "isMultifactorRequired" to this.isMultifactorRequired,
-            "isMultifactorEnrollRequired" to this.isMultifactorEnrollRequired,
-            "isMultifactorCodeInvalid" to this.isMultifactorCodeInvalid,
-            "isMultifactorTokenInvalid" to this.isMultifactorTokenInvalid,
-            "isPasswordNotStrongEnough" to this.isPasswordNotStrongEnough,
-            "isPasswordAlreadyUsed" to this.isPasswordAlreadyUsed,
-            "isRuleError" to this.isRuleError,
-            "isInvalidCredentials" to this.isInvalidCredentials,
-            "isRefreshTokenDeleted" to this.isRefreshTokenDeleted,
-            "isAccessDenied" to this.isAccessDenied,
-            "isTooManyAttempts" to this.isTooManyAttempts,
-            "isVerificationRequired" to this.isVerificationRequired,
-            "isNetworkError" to this.isNetworkError,
-            "isBrowserAppNotAvailable" to this.isBrowserAppNotAvailable,
-            "isPKCENotAvailable" to this.isPKCENotAvailable,
-            "isInvalidAuthorizeURL" to this.isInvalidAuthorizeURL,
-            "isInvalidConfiguration" to this.isInvalidConfiguration,
-            "isCanceled" to this.isCanceled,
-            "isPasswordLeaked" to this.isPasswordLeaked,
-            "isLoginRequired" to this.isLoginRequired,
-            "mfaToken" to this.getValue("mfa_token"),
-        )
-    )
+    var ex = this;
+    return buildMap<String, Any> {
+        put("_statusCode", ex.statusCode)
+        put("_errorFlags", mapOf(
+            "isMultifactorRequired" to ex.isMultifactorRequired,
+            "isMultifactorEnrollRequired" to ex.isMultifactorEnrollRequired,
+            "isMultifactorCodeInvalid" to ex.isMultifactorCodeInvalid,
+            "isMultifactorTokenInvalid" to ex.isMultifactorTokenInvalid,
+            "isPasswordNotStrongEnough" to ex.isPasswordNotStrongEnough,
+            "isPasswordAlreadyUsed" to ex.isPasswordAlreadyUsed,
+            "isRuleError" to ex.isRuleError,
+            "isInvalidCredentials" to ex.isInvalidCredentials,
+            "isRefreshTokenDeleted" to ex.isRefreshTokenDeleted,
+            "isAccessDenied" to ex.isAccessDenied,
+            "isTooManyAttempts" to ex.isTooManyAttempts,
+            "isVerificationRequired" to ex.isVerificationRequired,
+            "isNetworkError" to ex.isNetworkError,
+            "isBrowserAppNotAvailable" to ex.isBrowserAppNotAvailable,
+            "isPKCENotAvailable" to ex.isPKCENotAvailable,
+            "isInvalidAuthorizeURL" to ex.isInvalidAuthorizeURL,
+            "isInvalidConfiguration" to ex.isInvalidConfiguration,
+            "isCanceled" to ex.isCanceled,
+            "isPasswordLeaked" to ex.isPasswordLeaked,
+            "isLoginRequired" to ex.isLoginRequired,
+        ))
+        if (ex.getValue("mfa_token") != null) { put("mfa_token", ex.getValue("mfa_token")!!) }
```

Comment:
Let's use camelCase for the key.

Reply:
on iOS, this key is already present in `Auth0APIError.details` with key `mfa_token`, which is why i chose to do have the same key on the android side. I could replace this key in `auth0_flutter/ios/Classes/Extensions.swift:47`,

Reply:
Do you mean here https://github.com/auth0/auth0-flutter/blob/main/auth0_flutter/ios/Classes/Extensions.swift#L47?

Reply:
Yes, self.info` already contains the mfa token, but with `mfa_token` as key, I started with `mfaToken` until I tried testing the plugin on an iOS device.

Reply:
Yes, that's right.